man/iio_attr.1.in

```diff
@@ -84,7 +84,10 @@ Read and Write IIO Context attributes
 Read and Write IIO Debug attributes
 .TP
 .B \-S, \-\-Scan
-Scan for available IIO contexts, optional arg of specific backend(s) 'ip', 'usb' or 'ip:usb'. If no argument is given, it checks all that are availble.
+Scan for available IIO contexts, optional arg of specific backend(s) 'ip', 'usb' or 'ip:usb'.
+Specific options for USB include Vendor ID, Product ID to limit scanning to specific devices 'usb=0456,b673'.
```

Comment:
That would be `usb=0456:b673`

Reply:
we accept both `,` and ':` as pid/vid delimiters, but I think prefer ',' (less mangling). So that we what documented in the man pages.

Reply:
We must not support both. This will be extra-confusing, since Libiio will use commas for delimiting the backends in the list...